vlib/v/fmt/tests/if_keep.vv

```diff
@@ -0,0 +1,12 @@
+fn main() {
+	mut foo := 'foo'
+	mut bar := 'bar'
+	if true {
+		if foo == 'foo' {
+			foo = 'foo2'
+			bar = 'bar2'
+		}
+	}
+}
+
+// TODO: add more test cases
```

Comment:
It needs at least one case with a single statement in the then branch, one case with a single statement in the else branch, and one case with a single statement in both.